runtime-config-linux.md

```diff
@@ -0,0 +1,121 @@
+## Linux namespaces
+
+A namespace wraps a global system resource in an abstraction that makes it 
+appear to the processes within the namespace that they have their own isolated 
+instance of the global resource.  Changes to the global resource are visible to 
+other processes that are members of the namespace, but are invisible to other 
+processes. For more information, see [the man page](http://man7.org/linux/man-pages/man7/namespaces.7.html)
+
+Namespaces are specified in the spec as an array of entries. Each entry has a 
+type field with possible values described below and an optional path element. 
+If a path is specified, that particular file is used to join that type of namespace.
+
+```json
+    "namespaces": [
+        {
+            "type": "pid",
+            "path": "/proc/1234/ns/pid"
+        },
+        {
+            "type": "net",
```

Comment:
How about breaking it into two different structs? One for new namespaces and other for joining namespaces. Validation could check that there is no overlap for same namespace type in the two or there could be documented behavior on what has higher preference.

Sent from my iPhone

> On Aug 4, 2015, at 9:47 PM, Brandon Philips <notifications@github.com> wrote:
> 
> In runtime-config-linux.md:
> 
> > +instance of the global resource.  Changes to the global resource are visible to 
> > +other processes that are members of the namespace, but are invisible to other 
> > +processes. For more information, see [the man page](http://man7.org/linux/man-pages/man7/namespaces.7.html)
> > +
> > +Namespaces are specified in the spec as an array of entries. Each entry has a 
> > +type field with possible values described below and an optional path element. 
> > +If a path is specified, that particular file is used to join that type of namespace.
> > +
> > +```json
> > +    "namespaces": [
> > +        {
> > +            "type": "pid",
> > +            "path": "/proc/1234/ns/pid"
> > +        },
> > +        {
> > +            "type": "net",
> @mrunalp Any suggestions on how to resolve this?
> 
> —
> Reply to this email directly or view it on GitHub.
>